sparse/tests/test_core.py

```diff
@@ -153,6 +154,17 @@ def test_elemwise(func):
     assert_eq(func(x), func(s))
 
 
+@pytest.mark.parametrize('func', [operator.mul])
```

Comment:
Ah, I see, the current solution just appends and waits for future operations for force sum_duplicates.